spec/controllers/app/comments_controller_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe App::CommentsController, type: :controller do
+  let(:provider) { create(:user) }
+  let(:receiver) { create(:user) }
+  let(:feedback_session) { create(:feedback_session, provider: provider, receiver: receiver) }
+  let(:comment) { create(:comment, feedback_session: feedback_session) }
+
+  describe 'GET #index' do
+    before do
+      sign_in provider
+      get :index, params: { feedback_session_id: feedback_session.id }
+    end
+
+    it { expect(response).to have_http_status(:ok) }
+  end
+
+  describe "GET #new" do
+    before do
+      sign_in(provider)
+      get :new, params: { feedback_session_id: feedback_session.id }
+    end
+
+    it { expect(response).to have_http_status(:success) }
+  end
+
+  describe "POST #create" do
+    before do
+      sign_in(provider)
+      post :create, params: { feedback_session_id: feedback_session.id,  body: 'test'  }
+    end
+
+    it { expect(response).to have_http_status(:created) }
```

Comment:
igual acá podría ser bueno que agregues otro it que pruebe que realmente se creó, este it revisa que de el status http correcto, pero quizás verificar que se creó la entidad de alguna forma puede ser bueno

Reply:
Agregado el test!